packages/next/package.json

```diff
@@ -84,6 +84,7 @@
     "cacache": "13.0.1",
     "chokidar": "2.1.8",
     "css-loader": "3.5.3",
+    "etag": "1.8.1",
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/vercel/next.js/pull/12802/commits/7e854211b068a2536c3c17b0639b29032e324245